docs/connector-development/testing-connectors/connector-acceptance-tests-reference.md

```diff
@@ -36,27 +36,32 @@ Build your connector image if needed.
 docker build .
 ```
 
-Run one of the two scripts in the root of the connector:
+and test via one of the two following options
 
-- `python -m pytest -p integration_tests.acceptance` - to run tests inside virtual environment
+### (Prefered) Option 1: Run against the production acceptance test image
 
-  - On test completion, a log will be outputted to the terminal verifying:
+From the root of your connector run:
+```bash
+./acceptance-test-docker.sh
+```
 
-    - The connector the tests were ran for
-    - The git hash of the code used
-    - Whether the tests passed or failed
+This will run you local connector image against the same test suite that Airbyte uses in production
 
-    This is useful to provide in your PR as evidence of the acceptance tests passing locally.
+### (Debugging) Option 2: Run against the acceptance tests on your branch
 
-- `./acceptance-test-docker.sh` - to run tests from a docker container
+This will run the acceptance test suite directly with pytest. Allowing you to set breakpoints and debug your connector locally.
 
-If the test fails you will see detail about the test and where to find its inputs and outputs to reproduce it. You can also debug failed tests by adding `—pdb —last-failed`:
+The only pre-requisite is that you have [Poetry](https://python-poetry.org/docs/#installation) installed.
```

Comment:
Don't you need GSM credentials in `GCP_GSM_CREDENTIALS` ? In fact, I was wondering if it was even possible for third parties to run the acceptance tests locally, at all, considering that they don't have access to those secrets.